crates/next-api/src/pages.rs

```diff
@@ -926,28 +961,28 @@ impl PageEndpoint {
                     )
                     .await?;
 
-                let client_chunking_context =
-                    this.pages_project.project().client_chunking_context();
-                let dynamic_import_entries = collect_chunk_group(
-                    Vc::upcast(client_chunking_context),
-                    &next_dynamic_imports,
-                    Value::new(AvailabilityInfo::Root),
-                )
-                .await?
-                .to_resolved()
-                .await?;
-
                 let server_asset_trace_file = if this
                     .pages_project
                     .project()
                     .next_mode()
                     .await?
                     .is_production()
                 {
+                    let loadable_manifest_output =
+                        self.react_loadable_manifest(*dynamic_import_entries, NextRuntime::NodeJs);
```

Comment:
Is the next dynamic manifest not using in development?

Reply:
It is. This is only to get the filename to add to the NFT list

Further down in `fn output()` it is called again and added to the list of output assets.